docs/integrations/hosts-operating-systems/opentelemetry/opentelemetry-collector-insights.md

```diff
@@ -0,0 +1,447 @@
+:::
+
+## Fields creation in Sumo Logic for OpenTelemetry Collector Insights
+
+Following are the [fields](/docs/manage/fields/) which will be created as part of OpenTelemetry Collector Insights app installation, if not already present.
+
+- `sumo.datasource`. Has fixed value of `otel_collector`.
+- `_contentType`. Has fixed value of `OpenTelemetry`.
+- `deployment.environment`. User configured. Enter a name to identify your deployment environment.
+
+## Prerequisites
+
+### For OTLP endpoint configuration
+
+Before configuring the OTEL Collector integration, ensure you have the following prerequisites in place:
+
+1. **Sumo Logic OTLP Source**: You need to create an OTLP source in your Sumo Logic hosted collector. The OTLP source will provide the endpoint URL that the OTEL Collector will use to send telemetry data.
+
+  **Documentation**: [Creating a Sumo Logic OTLP Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/otlp/)
+
+### For metrics collection
+
+The OpenTelemetry Collector must be configured to export its own metrics using the built-in telemetry capabilities. This requires:
+- OpenTelemetry Collector version 0.130.1-sumo-0 or later
+- Collector configured with telemetry metrics enabled at `detailed` level
+- Access to OTLP endpoint for metrics export
+- Internal metrics exposed on port 8888 (default)
+
+### For logs collection
+
+The OpenTelemetry Collector must be configured to export its own logs using the built-in telemetry capabilities. This requires:
+- Collector configured with telemetry logs enabled at `debug` level
+- JSON encoding for structured log output
+- Access to OTLP endpoint for logs export
+
+### System Requirements
+
+- OTEL Collector v0.130.1-sumo-0 or later
+- Sufficient system resources (CPU, memory) for data processing
+- Proper permissions for the collector service to access configured resources
+
+import LogsCollectionPrereqisites from '../../../reuse/apps/logs-collection-prereqisites.md';
+
+<LogsCollectionPrereqisites/>
+
+## Collection configuration and app installation
+
+import ConfigAppInstall from '../../../reuse/apps/opentelemetry/config-app-install.md';
+
+<ConfigAppInstall/>
+
+### Step 1: Set up collector
+
+import SetupColl from '../../../reuse/apps/opentelemetry/set-up-collector.md';
+
+<SetupColl/>
+
+### Step 2: Configure integration
+
+OpenTelemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) YAML file with all the details concerning the data that needs to be collected.
+
+In this step, you will configure the OpenTelemetry Collector's built-in telemetry to monitor itself.
+
+The collector's service configuration needs to be updated to enable telemetry export. Below is the required configuration that should be added to your collector's service section:
```

Comment:
This seems to be confusing. Why are we calling it collectors service configuration ? Can we change this to make it more consistent with the format - similar to that of other opentelemetry app docs ?